oops/backplane/lighting.py

```diff
@@ -409,7 +409,8 @@ def lighting_test_suite(bpt):
 
         # Select the 95th percentile as a large but representative value
         # This is needed because km_per_fov_radian diverges near limb
-        values = deg_per_fov_radian.vals[deg_per_fov_radian.antimask]
+        values = deg_per_fov_radian[deg_per_fov_radian.antimask].vals
```

Comment:
This is still going to cause a problem because the result ot np.sort() has the same shape as the input, so it still needs to be flattened at some point before indexing it with the cutoff index.  This doesn't trip a unit test error in this pull request because the test that triggers it occurs inthe galileo ssi pull request.

Reply:
I'm changing this to np.sort(values, axis=None), because axis=None tells it to sort across all axes, yielding a 1-D array no matter the shape. Note that values.flatten() makes a copy first, which is unnecessary in this case. np.sort(values.ravel()) would be similar to axis=None, because ravel only makes a copy if it needs to.